pkg/util/helper/patch.go

```diff
@@ -32,3 +50,40 @@ func GenMergePatch(originalObj interface{}, modifiedObj interface{}) ([]byte, er
 
 	return patchBytes, nil
 }
+
+// GenReplaceFieldJSONPatch returns the RFC6902 JSONPatch array as []byte, which is used to simply
+// add/replace/delete certain JSON **Object** field.
+func GenReplaceFieldJSONPatch(path string, originalFieldValue, newFieldValue interface{}) ([]byte, error) {
```

Comment:
Can you help add some ut for this function?

Reply:
Done.